query.go

```diff
@@ -564,11 +565,10 @@ func queryDriverProperties(
 				continue
 			}
 			needToClose = append(needToClose, d)
+			m[d] = d.Properties()
 		} else {
 			logger.Debugw("driver not available", "name", d.Info().Name, "label", d.Info().Label, "status", status)
 		}
-
-		m[d] = d.Properties()
```

Comment:
We shouldn't get properties for unavailable drivers. This fixes a bug where to camera components may map to the same video path.